stubs/redis/redis/sentinel.pyi

```diff
@@ -39,5 +42,5 @@ class Sentinel(object):
     def discover_master(self, service_name): ...
     def filter_slaves(self, slaves): ...
     def discover_slaves(self, service_name): ...
-    def master_for(self, service_name, redis_class=..., connection_pool_class=..., **kwargs) -> None: ...
-    def slave_for(self, service_name, redis_class=..., connection_pool_class=..., **kwargs) -> None: ...
+    def master_for(self, service_name: str, redis_class: Type[_Redis] = ..., connection_pool_class=..., **kwargs) -> _Redis: ...
```

Comment:
Now we need to add overloads to handle the case where `redis_class` is not given:

```suggestion
    @overload
    def master_for(self, service_name: str, *, connection_pool_class=..., **kwargs) -> Redis[str]: ...
    @overload
    def master_for(self, service_name: str, redis_class: Type[_Redis] = ..., connection_pool_class=..., **kwargs) -> _Redis: ...
```

Otherwise, type checkers can't infer the type argument to `Redis` and would infer `Redis[Any]`.

(Same for `slave_for`)

Reply:
OK, Thanks :-)